protoc-gen-gogoroach/main.go

```diff
@@ -0,0 +1,41 @@
+// This is a customized version of https://github.com/gogo/protobuf/blob/master/protoc-gen-gogoslick/main.go
+
+package main
+
+import (
+	"github.com/gogo/protobuf/vanity"
+	"github.com/gogo/protobuf/vanity/command"
+)
+
+func main() {
+	req := command.Read()
+	files := req.GetProtoFile()
+
+	vanity.ForEachFile(files, vanity.TurnOnMarshalerAll)
```

Comment:
Is there any other reason for having our own protoc-gen than enabling these options? I'm not at all a fan of removing the options from the .proto files so that they can be centralized here. Doing so does nothing for the clarity of the code while making it marginally easier to add or change an option in the future which is a very rare operation.